ext/posix-spawn.c

```diff
@@ -283,6 +283,11 @@ each_env_i(VALUE key, VALUE val, VALUE arg)
 		const char *ev = envp[i];
 
 		if (strlen(ev) > name_len && !memcmp(ev, name, name_len) && ev[name_len] == '=') {
+            /* This operates on a duplicated environment -- release the
+             * existing entry memory before shifting the subsequent entry
+             * pointers down. */
```

Comment:
Leading whitespace looks off here?

Reply:
You are correct! This file uses tabs for indentation, oops. I'll re-push my branch to fix this up.